api-mocks/mappings/endpoints-mapping.json

```diff
@@ -368,6 +368,187 @@
         },
         "bodyFileName": "aem/v6.5.2/bundleinfo.json"
       }
+    },
+    {
+      "request": {
+        "method": "POST",
+        "url": "/zabbix/api_jsonrpc.php",
+        "bodyPatterns": [
+          {
+            "equalToJson": "{\n            \"jsonrpc\": \"2.0\",\n            \"method\": \"user.login\",\n            \"params\": {\n              \"user\": \"admin\",\n              \"password\": \"admin\"\n            },\n            \"id\": 1,\n        \"auth\": null  }"
+          }
+        ]
+      },
+      "response": {
+        "status": 200,
+        "headers": {
+          "Content-Type": "application/json"
+        },
+        "bodyFileName": "zabbix/auth.json"
+      }
+    },
+    {
+      "request": {
+        "method": "POST",
+        "url": "/zabbix/api_jsonrpc.php",
+        "bodyPatterns": [
+          {
+            "equalToJson": "{\n            \"jsonrpc\": \"2.0\",\n            \"method\": \"user.login\",\n            \"params\": {\n              \"user\": \"\",\n              \"password\": \"\"\n            },\n            \"id\": 1,\n        \"auth\": null  }"
+          }
+        ]
+      },
+      "response": {
+        "status": 200,
+        "headers": {
+          "Content-Type": "application/json"
+        },
+        "bodyFileName": "zabbix/auth_error.json"
+      }
+    },
+    {
+      "request": {
+        "method": "POST",
+        "url": "/zabbix/api_jsonrpc.php",
+        "bodyPatterns": [
+          {
+            "equalToJson": "{\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"item.get\",\n    \"params\": {\n        \"filter\": {\n            \"host\": [\n                \"\"\n            ]\n        },\n        \"search\": {\n            \"key_\": \"system.cpu.util[,idle]\"\n        },\n        \"sortfield\": \"name\"\n    },\n    \"id\": 1,\n    \"auth\": \"c2d2b213484b7f93e6639c1ae4e2e140\"\n}"
```

Comment:
Can we make this simpler so the matching uses only `system.cpu.util[,idle]`

Reply:
i believe that here i should use "equalToJson" which is match whole json body

Reply:
I mean like this:
```
    "bodyPatterns" : [ {
      "matchesJsonPath" : "$.params.search[?(@.key_ == 'system.cpu.util[,idle]')]"
    } ]
```

Reply:
yes, you are right, it's works. 
fixed